server/.env

```diff
@@ -0,0 +1,2 @@
+MAILCHIMP_API_KEY=b844b735d94a175b433b54a8a65bee83-us15
```

Comment:
Should not be committed, especially in a public repository; You'll need to revoke this key and reissue a new one